docs/pages/blog/2020-developer-survey-results.md

```diff
@@ -251,29 +279,32 @@ section.
 
 <img src="/static/blog/2020-survey/19.png" style="width: 728px; margin-top: 32px; margin-bottom: 24px;" alt="Pie chart: 53.84%	Material-UI styles (JSS), 20.41%	Styled components, 13.01%	Good plain CSS, 8.31%	CSS Modules, 1.96%	Emotion, 0.59%	scss, 0.59%	sass, 0.09%	less, 1.19%	Other" />
 
-The trends seems to be similar to the one from the last year's survey, so we will push with better support for styled components.
+The trend seems to be similar to the one from the last year's survey, so we will push with better support for styled components.
 
 ### 20. Has your organization ever paid for UI components?
 
 <img src="/static/blog/2020-survey/20.png" style="width: 553px; margin-top: 32px;" alt="Pie chart: 89.90%	No, 10.10% Yes" />
 
+
+
 ### 21. What type system are you using?
 
 <img src="/static/blog/2020-survey/21.png" style="width: 614px; margin-top: 32px;" alt="Bar chart: 54.08%	None
 30.87%	TypeScript 3.8, 7.31%	TypeScript 3.7, 1.90%	Flow, 1.55%	TypeScript 3.6, 0.98%	TypeScript 3.5, 3.31%	Other" />
 
+Almost half of the community is using TypeScript. Next year, it might even get more popular than JavaScript.
+
 ## Conclusion
 
 This data is **incredibly valuable** for our team.
 Thank you again for participating!
 We want to work on the problems that resonate the most with our users.
-[It's clear](#4-how-can-we-improve-material-ui-for-you) that we should:
+[It's clear](#5-how-can-we-improve-material-ui-for-you) that we should:
 
-1. Support more components
-2. Improve the customization
-3. Improve the documentation
-4. Improve the static typings
-5. Reduce the overhead of the library
+1. Increase the front-end user base by providing more flexibility on the components, unstyled components (pure hooks?).
```

Comment:
```suggestion
1. Increase the front-end user base by providing more flexibility on the components, unstyled components (pure hooks?).
```
Increasing the user base isn't something we should do to help the developer community.

Reply:
I don't follow:

- We benefit from a network effect. Each new developer using Material-UI adds value to the existing users. Most developers are well aware of this effect. One example of this is StackOverflow.
- We don't increase the userbase by saying we want to. We have to solve real problems to do such, isn't that a good thing? More developers finding shoes to their feet?

Reply:
If you read this from the perspective of a current or potential developer, not from that of the author, you'll hopefully see why this is the wrong motivation to attach these to. It's about DX, not growth. Growth is a secondary effect of improving the DX.

Reply:
Agree for focusing on the pain points

Reply:
Done